logs_sdk/lib/opentelemetry/sdk/logs/configuration_patch.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry/sdk/configurator'
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # The ConfiguratorPatch implements a hook to configure the logs portion
+      # of the SDK.
+      module ConfiguratorPatch
```

Comment:
I am curious, is it intentional the the module name is ConfiguratorPatch (with an r) but the filename is configuration_patch.rb (with an n) ?